package.json

```
@@ -28,7 +28,7 @@
     "generate-coverage": "./scripts/generate-coverage"
   },
   "dependencies": {
-    "commander": "5.1.0",
+    "commander": "^7.2.0",
```

```suggestion
    "commander": "7.2.0",
```

Better for package security and maintenance.